src/jit/rationalize.cpp

```diff
@@ -1903,31 +1667,6 @@ Compiler::fgWalkResult Rationalizer::SimpleTransformHelper(GenTree **ppTree, Com
         JITDUMP("\n");
         return SimpleTransformHelper(ppTree, data);
     }
-    else if (tree->gtOper == GT_QMARK)
-    {
-        // only certain forms of qmarks are allowed
-        // qmark(conditionExpr, 1, 0) is equivalent to conditionExpr
```

Comment:
I had thought that these were allowed in at least one of the backends that uses the Rationalizer, however, looking at the code it appears that we eliminate all qmarks, including the simple 1/0 ones.